configure.ac

```diff
@@ -246,6 +246,7 @@ OPTION_DEFAULT_DISABLE([yaml], [ENABLE_YAML])
 
 dnl Options for store
 OPTION_DEFAULT_ENABLE([store], [ENABLE_STORE])
+OPTION_DEFAULT_ENABLE([decomp], [ENABLE_DECOMP])
```

Comment:
In a previous review, you said that we want to be able to disable decomp build when there are no stores. So could this just use the ENABLE_STORE option? Does it really need its own?

Reply:
Added this to support a minimum build that could be installed on a network switch (slingshot switch in this case).

Reply:
I am sorry; I am not following. Are you saying that on network switches, you need to enable stores, but disable decomp?

If you want to disable both stores and decomp at the same time, then decomp could be disabled with the "ENABLE_STORE" variable, and wouldn't need its own.

Reply:
I see. You are suggesting changing the ENABLE store option for both. Will change and resubmit:)